single_kernel_mongo/config/statuses.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""File containing all possible statuses for Mongo* charms.
+
+TODO (Future PR(s)):
+- Add all statuses here
+- Update to be consistent with the spec
+
+Note: The structure of this file is subject to change, as the implementation spec is still in
+progress. However the idea that all statuses belong in one file holds true regardless of the spec.
+"""
+
+from enum import Enum
+
+from ops.model import BlockedStatus, WaitingStatus
+
+
+class MongoDB(Enum):
```

Comment:
Can we change the name to a more specific one?
```suggestion
class MongoDBStatuses(Enum):
```

Reply:
yes good idea!